docs/home/quickstart.md

```diff
@@ -1,70 +1,334 @@
 ---
-sidebar_position: 2
+sidebar_position: 3
 id: quickstart
 ---
 
-# Quickstart
- 
-## Step 1: Create your account
-
-[Register your account](https://dashboard.texturehq.com/sign-up) with a company or entity specific email address. We strongly recommend avoiding the use of personal email addresses. This helps us create a more secure experience and support associating you with your Organization.
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-![Sign Up Screenshot](https://i.giphy.com/fTnNtMJpeGMcjKxJVm.webp)
 
-## Step 2: Verify your email
+# Quickstart
 
-To ensure that you are not a bad actor, please check your inbox and verify your email address before creating or joining an organization.
+## Introduction
+This guide will walk you through the process of generating a link that allows your customers to connect their devices with Texture.
+
+## Generate a setup link
+
+There are three ways to generate a link:
+1. The Texture API
+2. The Texture SDK (Javascript, React, or React Native)
+3. The Texture Dashboard
+
+### Texture API and SDK
+
+#### Get Your API Key
+Go to the [Texture Dashboard Developer tab](https://dashboard.texturehq.com/developer) to obtain an API key
+
+#### Generate a link
+
+<Tabs>
+<TabItem value="js" label="API (Javascript)">
+
+Node v18 + required
+
+``` javascript
+#!/usr/bin/env node
+
+const textureApiUrl = "https://api.texture.energy";
+const apiKey = process.env.TEXTURE_API_KEY || "<your-api-key-goes-here>";
+
+console.log(`Using API key: ${apiKey}`);
+
+// Node v18 has fetch built in
+fetch("https://api.texture.energy/v1/connections", {
```

Comment:
```suggestion
fetch("https://api.texturehq.com/v1/connections", {
```

This is not your fault, this was taken from our open source examples (now fixed I believe) that pointed to an older url -- this will still work, but we're trying to get people to use the new `api.texturehq.com` url instead